crates/vrm-spec/src/serde_utils.rs

```diff
@@ -1,4 +1,4 @@
-/// This mod handles common issues when using VRMs.
+//! This mod handles common issues when using VRMs.
```

Comment:
Ref: https://doc.rust-lang.org/reference/comments.html

> Line comments beginning with //! and block comments /*! ... */ are doc comments that apply to the parent of the comment, rather than the item that follows.